api_inference_community/routes.py

```diff
@@ -94,26 +97,36 @@ def emit(self, record):
         # https://stackoverflow.com/questions/43344819/reading-response-headers-with-fetch-api/44816592#44816592
         "access-control-expose-headers": f"{HF_HEADER_COMPUTE_TYPE}, {HF_HEADER_COMPUTE_TIME}",
     }
+
     if status_code == 200:
         headers.update(**{k: str(v) for k, v in metrics.items()})
         task = os.getenv("TASK")
         if task == "text-to-speech":
-            # Special case, right now everything is flac audio we can output
             waveform, sampling_rate = outputs
-            data = ffmpeg_convert(waveform, sampling_rate)
-            headers["content-type"] = "audio/flac"
+
+            # Use the first valid mime type and its corresponding extension for the conversion format or "flac" as default
+            content_type, ext = list(mime_types.items())[0] if mime_types else ("audio/flac", "flac")
+            format_for_conversion = ext.lstrip(".")
+
+            data = ffmpeg_convert(waveform, sampling_rate, format_for_conversion)
+            headers["content-type"] = content_type
```

Comment:
This could simply become something like:

```python
format_for_conversion = params.get("audio_format", "flac")
data = ffmpeg_convert(waveform, sampling_rate, format_for_conversion)
headers["content-type"] = CONTENT_TYPE[format_for_conversion]
```
`CONTENT_TYPE` being a static map of ffmpeg names to content-types.